lib/api/objectPut.js

```diff
@@ -45,10 +44,19 @@ function _storeIt(bucketName, objectKey, objMD, authInfo, canonicalID,
         authInfo,
         metaHeaders,
         size,
-        contentType,
+        contentType: request.headers['content-type'],
+        cacheControl: request.headers['cache-control'],
+        contentDisposition: request.headers['content-disposition'],
+        expires: request.headers.expires,
         headers,
         log,
     };
+    // Remove V4 streaming value 'aws-chunked' if present in content-encoding to
```

Comment:
OK. Is it all right that I put the comment and reference on the utility function definition and removed it elsewhere, or should we keep the comments when calling the method?